src/components/MarkerInfo/BoxInfo/BoxInfo.jsx

```diff
@@ -115,29 +115,38 @@ const BoxInfo = ({
                 <FormLabel htmlFor="name" className={styles['form-label']}>
                   Name
                 </FormLabel>
-                <Input isReadOnly id="name" type="name" value={boxHolderName} />
+                {boxHolderName && <Input isReadOnly id="name" type="name" value={boxHolderName} />}
                 {/* Box email */}
                 <FormLabel isReadOnly htmlFor="email" className={styles['form-label']}>
                   Email
                 </FormLabel>
-                <Input isReadOnly id="email" type="email" value={boxHolderEmail} />
+                {boxHolderEmail && (
+                  <Input isReadOnly id="email" type="email" value={boxHolderEmail} />
+                )}
```

Comment:
If there is an empty field, then we shouldn't show the label for that field either. Can we make that edit here?

Reply:
Same with the rest of the fields for BoxInfo